src/renderer/features/fellowship-referendum-details/components/shared/ReferendumVoteChart.tsx

```diff
@@ -4,20 +4,21 @@ import { memo } from 'react';
 
 import { useI18n } from '@/shared/i18n';
 import { nullable } from '@/shared/lib/utils';
-import { FootnoteText } from '@/shared/ui';
-import { VoteChart } from '@/shared/ui-entities';
-import { Skeleton, Tooltip } from '@/shared/ui-kit';
-import { type Referendum, referendumService } from '@/domains/collectives';
+import { HelpText } from '@/shared/ui';
+import { DynamicVoteChart, VoteChart } from '@/shared/ui-entities';
+import { Box, Skeleton, Tooltip } from '@/shared/ui-kit';
+import { type Referendum, type Vote, referendumService } from '@/domains/collectives';
 import { fellowshipReferendumsDetailsFeature } from '../../model/feature';
 import { thresholdsModel } from '../../model/thresholds';
 
 type Props = {
   referendum: Referendum | null;
   pending: boolean;
   descriptionPosition: 'tooltip' | 'bottom';
+  votes?: Vote | null;
```

Comment:
`votes` > `Vote`, plural and singular looks strange, let's we come up with better naming